go/intro/extra-types-i/README.md

```diff
@@ -0,0 +1,14 @@
+name: Extra Types

```

Comment:
Would the name "Pointers, Structs, Arrays & Slices" work? 

Also, the description doesn't mention slices. Is that deliberate?

Reply:
It would